templates/rhpam73-authoring-ha.yaml

```diff
@@ -120,92 +120,92 @@ parameters:
   name: KIE_SERVER_MODE
   value: "DEVELOPMENT"
   required: false
 - displayName: KIE MBeans
-  description: KIE server mbeans enabled/disabled (Sets the kie.mbeans and kie.scanner.mbeans system properties)
+  description: KIE server mbeans enabled/disabled. (Sets the kie.mbeans and kie.scanner.mbeans system properties)
   name: KIE_MBEANS
   value: enabled
   required: false
 - displayName: Drools Server Filter Classes
-  description: KIE server class filtering (Sets the org.drools.server.filter.classes system property)
+  description: KIE server class filtering. (Sets the org.drools.server.filter.classes system property)
   name: DROOLS_SERVER_FILTER_CLASSES
   value: 'true'
   required: false
 - displayName: Business Central Custom http Route Hostname
-  description: 'Custom hostname for http service route.  Leave blank for default hostname,
-    e.g.: <application-name>-rhpamcentr-<project>.<default-domain-suffix>'
+  description: 'Custom hostname for http service route for Business Central. Leave blank for default
+    hostname, e.g.: <application-name>-rhpamcentr-<project>.<default-domain-suffix>'
   name: BUSINESS_CENTRAL_HOSTNAME_HTTP
   value: ''
   required: false
 - displayName: Business Central Custom https Route Hostname
-  description: 'Custom hostname for https service route.  Leave blank for default
+  description: 'Custom hostname for https service route for Business Central. Leave blank for default
     hostname, e.g.: secure-<application-name>-rhpamcentr-<project>.<default-domain-suffix>'
   name: BUSINESS_CENTRAL_HOSTNAME_HTTPS
   value: ''
   required: false
 - displayName: KIE Server Custom http Route Hostname
-  description: 'Custom hostname for http service route. Leave blank for default hostname,
+  description: 'Custom hostname for http service route for KIE Server. Leave blank for default hostname,
       e.g.: <application-name>-kieserver-<project>.<default-domain-suffix>'
   name: KIE_SERVER_HOSTNAME_HTTP
   value: ''
   required: false
 - displayName: KIE Server Custom https Route Hostname
-  description: 'Custom hostname for https service route.  Leave blank for default
+  description: 'Custom hostname for https service route for KIE Server. Leave blank for default
     hostname, e.g.: secure-<application-name>-kieserver-<project>.<default-domain-suffix>'
   name: KIE_SERVER_HOSTNAME_HTTPS
   value: ''
   required: false
 - displayName: Use the secure route name to link application requests to KIE_SERVER_HOSTNAME_HTTPS vs KIE_SERVER_HOSTNAME_HTTP.
-  description: 'If true, will use secure-APPLICATION_NAME-kieserver vs. APPLICATION_NAME-kieserver as the route name.'
+  description: 'If true, the KIE server will report secure-APPLICATION_NAME-kieserver vs. APPLICATION_NAME-kieserver as the route name in its API, including a connection to Business Central. Therefore, if this parameter is set to true, Business Central establishes a secure connection with the KIE server.'
```

Comment:
This is incorrect. It would be better stated like this:

"If true, the KIE server will use secure-APPLICATION_NAME-kieserver vs. APPLICATION_NAME-kieserver to lookup the KIE server Route for Business Central to report.'

We use websockets for connections between BC and KIE server, not REST. So from the user's perspective this change is all about which URL will show up in BC for them to click on when viewing the kie server list.

Reply:
@errantepiphany to clarify: this parameter does ont affect REST at all, only the websockets used for BC? So, except interaction with BC, the parameter affects nothing?

Reply:
@mramendi When I compare REST with Websockets, I am talking about the communication between BC and KIE Servers, not any kind of REST interface for users to hit KIE Servers with. With OpenShiftStartupStrategy, neither is used, and instead, a ConfigMap is used. Without OpenShiftStartupStrategy, then only Websockets is used, since we have hardcoded that in the templates. We do not allow the users to choose REST.

Reply:
This was resolved for description for now; not resolving in PR to allow for discussion between Davig and Evan.